test/Jamfile.v2

```diff
@@ -144,6 +129,35 @@ test-suite graph_test :
     [ run delete_edge.cpp ]
     ;
 
+alias graph_test_no_msvc14 : :
+        <toolset>msvc
+        <toolset-msvc:version>14.0
+        <address-model>64
+    ;
```

Comment:
I'm not that familiar with this alias business; does this alias include only msvc, except version 14.0? Or does it include all toolchains except msvc 14.0?

Reply:
It includes all toolchains except msvc 14.0.